README.md

```diff
@@ -45,10 +45,10 @@ This repository includes example applications written in both Swift and Objectiv
    * Swift Package Manager 5.3 or above
 * Xcode 11 or above (Xcode 12 or above if using Swift Package Manager)
```

Comment:
The CircleCI configuration is only building in Xcode 12.5.1, so we should make Xcode 12 the minimum supported Xcode version.